rtf_vis_tool/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "rtf-intro",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.8",
+    "@testing-library/react": "^11.2.3",
+    "@testing-library/user-event": "^12.6.0",
+    "drei": "^2.2.13",
+    "numjs": "https://github.com/nicolaspanel/numjs",
+    "quaternion": "^1.1.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
+    "react-spring": "^8.0.27",
+    "react-three-fiber": "^5.3.12",
+    "react-xarrows": "^1.5.2",
+    "styled-components": "^5.2.1",
+    "three": "^0.124.0",
+    "web-vitals": "^0.2.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  }
+}
```

Comment:
This file briefly lists the major dependencies that the users needs to have installed in order for the react app to run.

Reply:
look under "dependencies"

Reply:
Thanks for the explanation -- that helps a lot. Would you mind explaining in the readme why we need each of these dependencies (or at least what each group does)?

I know that's a bit tedious, but will make it a ton easier to maintain after the summer for react novices like myself

Reply:
yup. no problem at all!